### PR TITLE
[CI] Add GSM actions to public repos

### PR DESCRIPTION
**Pull request by serbantarmure**
### Description

Those actions are alreadi in `toptal/actions`

### How to test

- FIXME: Add the steps describing how to verify your changes

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/davinci-github-actions/blob/master/_docs/contribution/changeset-guidelines.md) (if needed)

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

### CODE REVIEW

**Comment by serbantarmure** [comment]
@toptal-anvil ping reviewers

File: expose-json-outputs/.tool-versions [hunk 1]
```diff
@@ -0,0 +1,2 @@
+yarn 1.17.3
+nodejs 16.16.0
```
Comment (dmaklygin) [inline expose-json-outputs/.tool-versions:2]: we've switched to node18 recently, I would use LTS version (18) instead

File: expose-json-outputs/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - uses: toptal/actions/expose-json-outputs@main
+    with:
+      json: ${{ steps.source.outputs.out }}
+  - name: Print foo
+    run: echo ${{ steps.expose.outputs.foo }}
+```
+
+### Inputs
+
+See [action.yml](action.yml).
```
Comment (dmaklygin) [inline expose-json-outputs/README.md:21]: we have a special command `yarn documentation:generate`. It generates README.md file automatically

File: expose-json-outputs/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (dmaklygin) [inline expose-json-outputs/package.json:1]: this repo is not a monorepo. Maybe, we will convert it to monorepo in a future, but now, all folders are just a folders.
this script (`index.js`) will be generated automatically. We run generator [here](https://github.com/toptal/davinci-github-actions/blob/cd532199698a4a60c2e2c6680a168afcced7bc4d/.lintstagedrc.js#L14)

**Review by Jakub-Kwak — APPROVED** [review]
LGTM!
However, I think we should also ask @toptal/secops for a review.

**Review by dmaklygin — APPROVED** [review]

**Review by cust0m — APPROVED** [review]

**Pull request merged**